test/fixtures/api/action-attributes.sourcemap.json

```diff
@@ -119,6 +119,19 @@
                       "content": [
                         {
                           "element": "object",
+                          "attributes": {
+                            "sourceMap": [
+                              {
+                                "element": "sourceMap",
+                                "content": [
+                                  [
+                                    73,
+                                    11
+                                  ]
+                                ]
+                              }
+                            ]
+                          },
```

Comment:
This is a source map for the `object` keyword in `+ Attributes` even though it is not defined.  I don't think we need a source map here.